app/pages/home/home-page.html

```diff
@@ -1,4 +1,4 @@
-<landing-page base-project-url="{{$ctrl.baseProjectUrl}}">
+<landing-page base-project-url="{{$ctrl.baseProjectUrl}}" show-header="$ctrl.saasOfferings.length" collapsed-header-text="Show SAAS Offerings">
```

Comment:
> I would not refer to these as SaaS Offerings in any UI text

+1

Additionally, I find the "Show..." link has visual affordances of a dropdown and/or a toggle for the catalog below (i.e., it feels like a control for the catalog, not for the now collapsed offerings area) because of the hard lines above and below the link.

![screen shot 2017-06-28 at 3 21 39 pm](https://user-images.githubusercontent.com/895728/27656184-0a55e1f4-5c16-11e7-853a-a3616893ed4e.PNG)

When the offerings are present, I that's not the case.

![screen shot 2017-06-28 at 3 22 26 pm](https://user-images.githubusercontent.com/895728/27656270-5290c858-5c16-11e7-8cfb-fab727454183.PNG)